package.json

```diff
@@ -36,9 +36,12 @@
         "ts-jest": "25.5.1",
         "tslint": "6.1.2",
         "tslint-config-prettier": "1.18.0",
+        "ts-loader": "8.0.4",
         "typedoc": "^0.17.6",
         "typedoc-plugin-markdown": "^2.2.17",
-        "typescript": "3.8.3"
+        "typescript": "3.8.3",
+        "webpack": "4.44.2",
+        "webpack-cli": "^3.3.12"
```

Comment:
Nitpick: change to explicit version

Reply:
Should I lock in these while I'm at it?...

```
"typedoc-plugin-markdown": "^2.2.17",
"@types/faker": "^4.1.12",
```